src/strands/event_loop/event_loop.py

```diff
@@ -95,6 +102,7 @@ async def event_loop_cycle(agent: "Agent", invocation_state: dict[str, Any]) ->
     attributes = {"event_loop_cycle_id": str(invocation_state.get("event_loop_cycle_id"))}
     cycle_start_time, cycle_trace = agent.event_loop_metrics.start_cycle(attributes=attributes)
     invocation_state["event_loop_cycle_trace"] = cycle_trace
+    output_schema: OutputSchema = invocation_state.get("output_schema")
```

Comment:
Again, we need to plumb through whatever parameters we need; invocation_state should not be modified